packages/listview/src/components/FloatingSearchButtonIcon/styled.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+import { CircleFilled } from '@rainbow-modules/icons';
```

Comment:
here instead of add an icon for this, use a span with a background and border-radius to achieve this